packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -593,6 +648,31 @@ const Conversation = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Get url from convo object. If there isn't one, get it from the cache
+   *
+   * @param {Object} convo convo object with either a url or id
+   * @returns {String} url for the specific convo
+   */
+  getConvoUrl(convo) {
+    if (convo.url) {
+      return convo.url;
```

Comment:
Does it make sense to update the cache here? It may not have been set earlier.

Reply:
That's a good idea. If the user passes in an `id` and a `url`, we could update the cache